packages/amplify-velocity-template/.gitignore

```diff
@@ -7,3 +7,4 @@ build
 .idea/*
 dist/
 coverage
+src/parse/index.js
```

Comment:
Will this work if not published?

Reply:
This file gets generated by gison at build time. we don't need to track this